### DISCUSSION TITLE
Night weather icons

### DISCUSSION THREAD

**Discussion by SashaCX75** [zepp-health/zeppos-forum#99] [Developer Feedback]
To display the weather you need 29 icons. The last three icons should show the weather at night. But they never really show up, not even at night.

**Comment by MyoungXUE** [comment]
Hi, thank you for your feedback, this is suspected to be a bug, I will feedback this issue to the relevant colleagues.

**Reply by MyoungXUE** [reply]
Hi, I just got a reply from my colleague, the system does not support the night weather icon for now, the related function will be supported in Zepp OS 3.0.

**Reply by idimas13** [reply]
This works even on older non-zeppos devices (GT2 series) already more than two years..... But not on flagships.
